R/utils.R

```diff
@@ -189,25 +189,46 @@ check_filter_datanames <- function(filters, datanames) {
 #' @param modules (`teal_modules`) object
 #' @param filters (`teal_slices`) object
 #' @param filtered_data_singleton A result of `teal_data_to_filtered_data` applied to `data`.
+#' @param progress (`Progress`) object, optional; will be filled
```

Comment:
@chlebowa what do you mean by `will be filled`?

Reply:
The progress bar will be filled during the (possibly recursive) funciton call.

Reply:
And here is why I didn't:
`Error in `initialize(...)`: Can only use Progress$new() inside a Shiny app`

Reply:
hmm, so maybe `(Progress) an optional progress object to be filled`

Reply:
I expanded the argument description to 
```
#' @param progress (`Progress`) object from `shiny`, optional.
#'  The progress bar will be filled during the (possibly recursive) call.
```

Reply:
amazing! thanks